utils/test_settings.py

```diff
@@ -28,3 +29,17 @@
     'DB': 'autoreduction',
     'PORT': '3306',
 }
+
+PATH_TO_FILE = os.path.dirname(os.path.realpath(__file__))
+PATH_TO_TEST_OUTPUT = os.path.join(PATH_TO_FILE, 'test-output')
+
+try:
+    os.makedirs(PATH_TO_TEST_OUTPUT)
```

Comment:
Do we want to place logic like this in `test_settings`? Having side effects of importing data structures could cause problems down the line

Reply:
You are right. Have you got a better suggestion for dealing with this I'd prefer not to use hard coded file paths if we can avoid it

Reply:
It could be the setup / tear down methods in unit tests which are responsible?
Then this just returns a path which is generated at runtime.